lib/mutant/mutator/node/begin.rb

```diff
@@ -13,23 +13,11 @@ class Begin < self
         #
         # @return [undefined]
         def dispatch
-          Util::Array.each(children, self, &method(:emit_child_subset))
           children.each_with_index do |child, index|
             mutate_child(index)
             emit(child) unless children.eql?([child])
```

Comment:
@dkubb The reason is not existend anymore as unparser was fixed for that weird edge case using `#one?`.